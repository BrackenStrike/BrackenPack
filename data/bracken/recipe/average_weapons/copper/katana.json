{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:wayfinder_armor_trim_smithing_template",
	"base": "minecraft:stone_sword",
	"addition": "minecraft:copper_ingot",
	"result": {
		"id": "minecraft:stone_sword",
		"components": {
			"minecraft:max_damage": 175,
			"minecraft:item_name": "Copper Katana",
			"minecraft:item_model": "bracken:average_weapons/copper/katana",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "5.5 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+5% Movement Speed","color":"dark_green","italic":false},
				{"translate": "Limited Jump Strength","color":"red","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:copper_katana",
					"amount": 4.5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:copper_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:copper_katana",
					"amount": 0.05,
					"operation": "add_value",
					"slot": "hand"
				},{
					"type": "minecraft:jump_strength",
					"id": "bracken:copper_katana",
					"amount": -0.5,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}