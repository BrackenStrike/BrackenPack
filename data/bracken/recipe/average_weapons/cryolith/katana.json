{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:wayfinder_armor_trim_smithing_template",
	"base": "minecraft:iron_sword",
	"addition": "minecraft:polar_bear_spawn_egg",
	"result": {
		"id": "minecraft:iron_sword",
		"components": {
			"minecraft:max_damage": 200,
			"minecraft:item_name": "Cryolith Katana",
			"minecraft:item_model": "bracken:average_weapons/cryolith/katana",
			"minecraft:enchantments": {
				"minecraft:vanishing_curse": 1,
				"bracken:freezing_fix": 1,
				"bracken:fragility_curse": 1
			},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "10 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+5% Movement Speed","color":"dark_green","italic":false},
				{"translate": "Limited Jump Strength","color":"red","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:cryolith_katana",
					"amount": 9,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:cryolith_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:cryolith_katana",
					"amount": 0.05,
					"operation": "add_value",
					"slot": "hand"
				},{
					"type": "minecraft:jump_strength",
					"id": "bracken:cryolith_katana",
					"amount": -0.5,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}