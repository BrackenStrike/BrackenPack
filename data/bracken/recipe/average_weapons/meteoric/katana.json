{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:wayfinder_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:netherite_scrap",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Meteoric Katana",
			"minecraft:item_model": "bracken:average_weapons/meteoric/katana",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "8 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+5% Movement Speed","color":"dark_green","italic":false},
				{"translate": "Limited Jump Strength","color":"red","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:meteoric_katana",
					"amount": 7,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:nmeteoric_katana",
					"amount": -3,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:meteoric_katana",
					"amount": 0.05,
					"operation": "add_value",
					"slot": "hand"
				},{
					"type": "minecraft:jump_strength",
					"id": "bracken:meteoric_katana",
					"amount": -0.5,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}