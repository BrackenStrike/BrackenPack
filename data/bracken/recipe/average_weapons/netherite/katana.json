{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:wayfinder_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:netherite_ingot",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "\"Netherite Katana\"",
			"minecraft:item_model": "bracken:average_weapons/netherite/katana",
			"minecraft:lore": [
				"{\"translate\":\"\",\"color\":white,\"italic\":false}",
				"{\"translate\":\"When in Main Hand:\",\"color\":gray,\"italic\":false}",
				"{\"translate\":\" 8 Attack Damage\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" 1.6 Attack Speed\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" +5% Movement Speed\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" Limited Jump Strength\",\"color\":red,\"italic\":false}"
			],
			"minecraft:attribute_modifiers": {
				"modifiers": [{
					"type": "minecraft:attack_damage",
					"id": "bracken:netherite_katana",
					"amount": 7,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:netherite_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:netherite_katana",
					"amount": 0.05,
					"operation": "add_value",
					"slot": "hand"
				},{
					"type": "minecraft:jump_strength",
					"id": "bracken:netherite_katana",
					"amount": -0.5,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"show_in_tooltip": false
			}
		}
	}
}