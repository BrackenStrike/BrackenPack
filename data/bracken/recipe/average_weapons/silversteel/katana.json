{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:wayfinder_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:wolf_spawn_egg",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Silversteel Katana",
			"minecraft:enchantments": {
				"bracken:silvered_fix": 1
			},
			"minecraft:enchantment_glint_override": false,
			"minecraft:item_model": "bracken:average_weapons/silversteel/katana",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "9 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+5% Movement Speed","color":"dark_green","italic":false},
				{"translate": "Limited Jump Strength","color":"red","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:silversteel_katana",
					"amount": 8,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:silversteel_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:silversteel_katana",
					"amount": 0.05,
					"operation": "add_value",
					"slot": "hand"
				},{
					"type": "minecraft:jump_strength",
					"id": "bracken:silversteel_katana",
					"amount": -0.5,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}