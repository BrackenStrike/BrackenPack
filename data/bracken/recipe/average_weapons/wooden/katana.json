{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:wayfinder_armor_trim_smithing_template",
	"base": "minecraft:wooden_sword",
	"addition": "#minecraft:planks",
	"result": {
		"id": "minecraft:wooden_sword",
		"components": {
			"minecraft:item_name": "\"Wooden Katana\"",
			"minecraft:item_model": "bracken:average_weapons/wooden/katana",
			"minecraft:lore": [
				"{\"translate\":\"\",\"color\":white,\"italic\":false}",
				"{\"translate\":\"When in Main Hand:\",\"color\":gray,\"italic\":false}",
				"{\"translate\":\" 4 Attack Damage\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" 1.6 Attack Speed\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" +5% Movement Speed\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" Limited Jump Strength\",\"color\":red,\"italic\":false}"
			],
			"minecraft:attribute_modifiers": {
				"modifiers": [{
					"type": "minecraft:attack_damage",
					"id": "bracken:wooden_katana",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:wooden_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:wooden_katana",
					"amount": 0.05,
					"operation": "add_value",
					"slot": "hand"
				},{
					"type": "minecraft:jump_strength",
					"id": "bracken:wooden_katana",
					"amount": -0.5,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"show_in_tooltip": false
			}
		}
	}
}