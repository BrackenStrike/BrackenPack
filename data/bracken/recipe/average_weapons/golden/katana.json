{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:wayfinder_armor_trim_smithing_template",
	"base": "minecraft:golden_sword",
	"addition": "minecraft:gold_ingot",
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:item_name": "\"Golden Katana\"",
			"minecraft:item_model": "bracken:average_weapons/golden/katana",
			"minecraft:lore": [
				"{\"translate\":\"\",\"color\":white,\"italic\":false}",
				"{\"translate\":\"When in Main Hand:\",\"color\":gray,\"italic\":false}",
				"{\"translate\":\" 4 Attack Damage\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" 1.6 Attack Speed\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" +5% Movement Speed\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" Limited Jump Strength\",\"color\":red,\"italic\":false}"
			],
			"minecraft:attribute_modifiers": {
				"modifiers": [{
					"type": "minecraft:attack_damage",
					"id": "bracken:golden_katana",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:golden_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:golden_katana",
					"amount": 0.05,
					"operation": "add_value",
					"slot": "hand"
				},{
					"type": "minecraft:jump_strength",
					"id": "bracken:golden_katana",
					"amount": -0.5,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"show_in_tooltip": false
			}
		}
	}
}